code/datums/ai/basic_mobs/basic_ai_behaviors/stop_and_stare.dm

```diff
@@ -0,0 +1,29 @@
+/// Makes a mob simply stop and stare at a movable... yea...
+/datum/ai_behavior/stop_and_stare
+	action_cooldown = 5 SECONDS // pretty much a placeholder
+	behavior_flags = AI_BEHAVIOR_MOVE_AND_PERFORM
+
+/datum/ai_behavior/stop_and_stare/setup(datum/ai_controller/controller, target_key)
+	. = ..()
+	var/datum/weakref/weak_target = controller.blackboard[target_key]
+	var/atom/movable/target = weak_target?.resolve()
+	return ismovable(target) && isturf(target.loc) && ismob(controller.pawn)
+
+/datum/ai_behavior/stop_and_stare/perform(seconds_per_tick, datum/ai_controller/controller, target_key)
+	// i don't really like doing this but we wanna make sure that the cooldown is pertinent to what we need for this specific controller before we invoke parent
+	action_cooldown = controller.blackboard[BB_STATIONARY_COOLDOWN]
+	. = ..()
+	var/datum/weakref/weak_target = controller.blackboard[target_key]
+	var/atom/movable/target = weak_target?.resolve()
+	if(!ismovable(target) || !isturf(target.loc)) // just to make sure that nothing funky happened between setup and perform
+		return
+
+	var/mob/pawn_mob = controller.pawn
+	var/turf/pawn_turf = get_turf(pawn_mob)
+
+	pawn_mob.face_atom(target)
+	pawn_mob.balloon_alert_to_viewers("stops and stares...")
```

Comment:
You don't have to do anything now, but in the future what I want us to use is a form of `manual_emote` that performs the Runechat animation, but doesn't put it in the chat.